# 问题：能回测不能模拟交易，网络错误

2015-10-08 20:04 [SYS ERROR] 
handle_data函数运行失败，今天无法生成信号: 

handle_data() internal error line 94, in handle_data File "python2.7/site-packages/DataAPI/DATAYES.py", 

line 928, in MktEqudGet raise Exception((u'%s for request: %s' % (csvString if csvString is not None else 'Network Error', ''.join(requestString))).encode('utf-8')) Exception: -1:No Data Returned for request: /api/market/getMktEqud.csv?appname=mercury&secID=002703.XSHE&ticker=&tradeDate=20151009&beginDate=&endDate=&field=openPrice

不知道能解决不？